whotracksme/data/assets/trackerdb.sql

```diff
@@ -3287,6 +3430,81 @@ INSERT INTO "trackers" VALUES('userzoom.com','UserZoom',3,'https://www.userzoom.
+INSERT INTO "trackers" VALUES('acint.net','Acint',6,'https://www.acint.net/','acint',NULL,NULL);
+INSERT INTO "trackers" VALUES('adalyser.com','Adalyser',6,'https://www.adalyser.com/','onesoon',NULL,NULL);
+INSERT INTO "trackers" VALUES('adblade.com','Adblade',4,'https://adblade.com/','adblade','385',NULL);
+INSERT INTO "trackers" VALUES('adkontekst.pl','Adkontekst',4,'http://netsprint.eu/','netsprint',NULL,NULL);
+INSERT INTO "trackers" VALUES('admitad.com','Admitad',4,'https://www.admitad.com/en/#','admitad','1306',NULL);
+INSERT INTO "trackers" VALUES('affec.tv','Affec.tv',4,'https://affectv.com/','affectv','1468',NULL);
+INSERT INTO "trackers" VALUES('airpr.com','AirPR',6,'https://airpr.com/','airpr',NULL,NULL);
+INSERT INTO "trackers" VALUES('allo-pages.fr','Allo-Pages',2,'http://www.allo-pages.fr/','links_lab',NULL,NULL);
+INSERT INTO "trackers" VALUES('apa.at','Apa',8,'http://www.apa.at/Site/index.de.html','apa',NULL,NULL);
+INSERT INTO "trackers" VALUES('artlebedev.ru','Art.Lebedev',8,'https://www.artlebedev.ru/','art.lebedev_studio',NULL,NULL);
+INSERT INTO "trackers" VALUES('atlassian.com','Atlassian',2,'https://www.atlassian.com/','atlassian',NULL,NULL);
+INSERT INTO "trackers" VALUES('awin1.com','Awin1',4,'https://www.awin.com/de','awin','1548','');
+INSERT INTO "trackers" VALUES('bannerflow.com','Bannerflow',4,'https://www.bannerflow.com/','bannerflow',NULL,NULL);
+INSERT INTO "trackers" VALUES('beeketing.com','Beeketing',6,'https://beeketing.com/','beeketing','2395',NULL);
+INSERT INTO "trackers" VALUES('bemobile.ua','Bemobile',10,'http://bemobile.ua/en/','bemobile',NULL,NULL);
+INSERT INTO "trackers" VALUES('betweendigital.com','Between Digital',6,'http://betweendigital.ru/ssp','between_digital','1488',NULL);
+INSERT INTO "trackers" VALUES('bid.run','Bid.Run',6,'http://bid.run/','bid.run','3024',NULL);
+INSERT INTO "trackers" VALUES('bigcommerce.com','BigCommerce',6,'https://www.bigcommerce.com/','bigcommerce',NULL,NULL);
+INSERT INTO "trackers" VALUES('blogfoster.com','Blogfoster',8,'http://www.blogfoster.com/','blogfoster',NULL,NULL);
+INSERT INTO "trackers" VALUES('btttag.com','Btttag',6,'https://www.bluetriangletech.com/','blue_triangle_technologies_inc','2260',NULL);
+INSERT INTO "trackers" VALUES('bugherd.com','BugHerd',2,'https://bugherd.com','macropod_software','1083',NULL);
+INSERT INTO "trackers" VALUES('cbsi.com','CBS Interactive',8,'https://www.cbsinteractive.com/','cbs_interactive','357',NULL);
+INSERT INTO "trackers" VALUES('cdnvideo.com','CDNvideo',9,'https://www.cdnvideo.com/','cdnvideo',NULL,NULL);
+INSERT INTO "trackers" VALUES('clearbit.com','Clearbit',6,'https://clearbit.com/','clearbit',NULL,NULL);
+INSERT INTO "trackers" VALUES('cnetcontent.com','Cnetcontent',8,'http://cnetcontent.com/','cbs_interactive',NULL,NULL);
+INSERT INTO "trackers" VALUES('cnzz.com','CNZZ',6,'http://www.umeng.com/','umeng','704','CNZZ is now Umeng');
```

Comment:
If the change of name was just a rebrand, use the new brand name for the tracker.

Reply:
Done!